### ISSUE TITLE
Support PyUSB & HIDAPI

### ISSUE THREAD

**Issue by JnyJny** [JnyJny/blynclight#11]
I know I've vacillated between using PyUSB and HIDAPI to access the hardware, the changes motivated by software availability/accessibility on my primary development platform (Mac OS). It occurs to me that it doesn't have to be an either/or choice, I could support either method of access.
